### PR TITLE
Alpha content review

### PR DESCRIPTION
**Pull request by bradmorton1**
## Summary
* Routine Change

Changes highlighted in the content review and json examples

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: specification/nhs-website-content-api.yaml [hunk 1]
```diff
@@ -18,8 +18,8 @@ info:
       * information about NHS services
 
     You cannot currently use this API to:
-      * access data from NHS Services
-      * access Ratings and Reviews of NHS organisations submitted to the NHS website
+      * [find services near you](https://www.nhs.uk/nhs-services/services-near-you/)
+      * [check ratings and reviews of organisations](https://developer.api.nhs.uk/nhs-api/documentation/comments)
```
Comment (adrianparr) [inline specification/nhs-website-content-api.yaml:22]: The 'Rating and Reviews' API is going to be moved to APIM soon too, not sure linking to the (old) Developer Portal is a good idea.

File: specification/nhs-website-content-api.yaml [hunk 2]
```diff
@@ -30,40 +30,24 @@ info:
       - Ratings and Reviews API
 
     ## API status and roadmap
-    This API has been moved from the nhs.uk developer API website. It is currently in Alpha phase. If you are an existing user, you need to sign up for the API in the new API Platform.
+    This API has moved from the nhs.uk developer API website. It is currently in development. If you are an existing user, you need to sign up for the API in the new API platform.
```
Comment (adrianparr) [inline specification/nhs-website-content-api.yaml:33]: This API has moved from the NHS website developer portal. It is currently in development. If you are an existing user, you need to create an account in the [Developer hub](https://digital.nhs.uk/developer).

File: specification/nhs-website-content-api.yaml [hunk 3]
```diff
@@ -30,40 +30,24 @@ info:
...
+    Page objects conform to global [schema.org](https://schema.org/) standards.
 
     ## Network access
     No special network access is required for this API. It is available over the internet.
 
     ## Security and authorisation
     This API is [application-restricted](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation#application-restricted-apis), meaning we authenticate the calling application but not the end user.
 
-    The end user will need to provide a API key in the header or as a query parameter in requests.
+    You must provide a API key in the header or as a query parameter in requests.
```
Comment (adrianparr) [inline specification/nhs-website-content-api.yaml:50]: "or as a query parameter in requests"
Will this still be possible?

File: specification/nhs-website-content-api.yaml [hunk 4]
```diff
@@ -82,33 +66,14 @@ info:
     You need to sign up via the API platform to recieve your key.
```
Comment (adrianparr) [inline specification/nhs-website-content-api.yaml:66]: You need to create an account in the Developer hub to get an API key.

File: specification/nhs-website-content-api.yaml [hunk 5]
```diff
@@ -259,12 +228,13 @@ paths:
                 $ref: components/examples/Medicines.json
   /medicines/*:
```
Comment (adrianparr) [inline specification/nhs-website-content-api.yaml:229]: Should the root 'Medicines A to Z' (/medicines/) endpoint be mentioned here too?

File: specification/nhs-website-content-api.yaml (hunk 1 above)
Comment (bradmorton1) [reply specification/nhs-website-content-api.yaml:22]: Sorry, that should link to the nhs.uk page

File: specification/nhs-website-content-api.yaml (hunk 3 above)
Comment (bradmorton1) [reply specification/nhs-website-content-api.yaml:50]: I think so but I can remove it and then ask in the design review

File: specification/nhs-website-content-api.yaml (hunk 5 above)
Comment (bradmorton1) [reply specification/nhs-website-content-api.yaml:229]: The root page is above

File: specification/nhs-website-content-api.yaml (hunk 1 above)
Comment (bradmorton1) [reply specification/nhs-website-content-api.yaml:22]: I might just remove it entirely

**Pull request merged**